visualize.rst

```diff
@@ -0,0 +1,188 @@
+**********************************************************
+Visualize Data with Google Fusion Tables and Google Earth
+**********************************************************
+
+The importance of location data is continuously increasing, so do the ways through which we can visualize this information. This guide helps the users to visualize the collected data uploaded on ODK Aggregate server using Google Fusion Tables and Google Earth.
+
+.. _visual-prerequisites:  
+
+Prerequisites
+~~~~~~~~~~~~~~~~
```

Comment:
Your headline hierarchy is not right.

Rst does not define a headline-character hierarchy. The one we specify in the contrib guide is conventional to our docs, but not built-in. Consequently, if you use a different underlining char (the `~` here), it becomes the next-level headline. 

So, your document hierarchy currently looks like:

 - Visualize Data with Google Fusion Tables and Google Earth
      - Prerequisites
             - Google Fusion Tables
             - Google Earth


Prereqs should probably be a rubric headline, and the tool-based section headlines should be double-underline (equal-sign: `====`)